<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportZySingle</class>
 <widget class="QDialog" name="ImportZySingle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>102</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导入掌阅单本书书单</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LabelXlsx">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Excel文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditXlsx"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonScan">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LabelSheet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>选择Sheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ComboBoxSheet"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOk">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ImportZySingle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ImportZySingle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
